microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md

```diff
@@ -28,13 +28,9 @@ The host or debuggee machine is the Windows 10 or later device that you want to
 
 ### Install and configure Microsoft Edge
 
-If you have not already, install Microsoft Edge from [this page](https://www.microsoft.com/edge).  If you are using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
+If you haven't already, install Microsoft Edge from [this page](https://www.microsoft.com/edge).  If you are using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge (Chromium) and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
 
-Now navigate to `edge://flags` in Microsoft Edge.  In **Search flags**, type in **Enable remote debugging through Windows Device Portal**.  Set that flag to **Enabled**.  Then, choose the **Restart** button to restart Microsoft Edge.
```

Comment:
@bwalderman - I removed this step (but kept the image).  Should this step in this section be retained?

Reply:
This is what I don't understand about GitHub PR UI: When I update a Comment on **Files Changed** tab, it doesn't update here in the **Conversations** tab.  I retained, not removed, this step.

Reply:
LGTM